tests/src/Integration/Action/EntityCreateTest.php

```diff
@@ -143,7 +143,7 @@ public function testRefiningContextDefinitions() {
     $this->action->refineContextDefinitions();
     $this->assertEquals(
       $this->action->getProvidedContextDefinition('entity')
-        ->getDataType(), 'entity:test:bundle_test'
+        ->getDataType(), 'test:bundle_test'
```

Comment:
this is wrong, the provided data type must be entity:test:bundle_test. We should only change the plugin ID.